src/app/shared/models/social-link.component.ts

```diff
@@ -0,0 +1,3 @@
+export class SocialLink {
+  constructor(public name: string, public link: string) {}
+}
```

Comment:
As I told you, the name of this file should be: social-link.model.ts, please change that